### PR TITLE
Makes mechs unable to drill space

### LINKED ISSUE #9114: Ripley drilling itself
If you are /somehow autistic enough/ to drill space, if you are on a space tile, in a ripley, you end up drilling yourself.

### PR DESCRIPTION
**Pull request by kingofkosmos**
Added a check so mechs aren't able to drill space. Thanks to Miauw for showing me how to do this!

Fixes #9114

### CODE REVIEW

**Comment by Jordie0608** [comment]
Not necessarily a problem, but this will stop mechs from being able to drill unsimulated turfs.

**Comment by Firecage** [comment]
Same goes for shuttle turfs, since those are neither simulated nor unsimulated.

**Comment by Aranclanos** [comment]
unsimulated turfs aren't supported and should be removed since linda was added. WIth the exception of space turfs, who are unsimulated.

File: code/game/mecha/equipment/tools/tools.dm [hunk 1]
```diff
@@ -87,6 +87,8 @@
 /obj/item/mecha_parts/mecha_equipment/tool/drill/action(atom/target)
 	if(!action_checks(target))
 		return
+	if(istype(target, /turf) && !istype(target, /turf/simulated))
```
Comment (xxalpha) [inline code/game/mecha/equipment/tools/tools.dm]: I think just !istype(target, /turf/simulated) is sufficient.

**Comment by Jalleo** [comment]
Does tgat meab we can have a giant simulated centcom?

**Comment by Aranclanos** [comment]
yes, thats what it SHOULD be. Waiting for someone to do it.

**Comment by Crystalwarrior** [comment]
Wouldn't simulated centcom cause unnecessary CPU usage?

**Comment by Aranclanos** [comment]
No

**Pull request merged**